broker/config_manager.py

```diff
@@ -40,6 +41,11 @@ class ConfigManager:
     """
 
     interactive_mode = sys.stdin.isatty()
+
+    if "BROKER_INTERACTIVE_MODE" in os.environ:
+        env_val = os.environ["BROKER_INTERACTIVE_MODE"].lower()
+        if env_val in ("false", "0", "no"):
+            interactive_mode = False
```

Comment:
If we instead treat the environment variable like a flag, then we can greatly simplify the check.
```suggestion
interactive_mode = os.environ.get("BROKER_INTERACTIVE_MODE", sys.stdin.isatty())
```

Reply:
will make the change! BTW, can you backport this to older versions, so that I can use it?